openslides/account/widgets.py

```diff
@@ -1,42 +0,0 @@
-from django.contrib.auth.models import AnonymousUser
-from django.utils.translation import ugettext_lazy
-
-from openslides.utils.personal_info import PersonalInfo
-from openslides.utils.widgets import Widget
-
-
-class PersonalInfoWidget(Widget):
```

Comment:
Could you please add an issue to discuss and decide whether we want to have something like a personal info widget or box in the new AngularJS app? Here we also should decide whether the server should provide a special read-only REST view with specific data (my motions, where I am on the list of speakers ...).